docs/content/technical-specifications.md

```diff
@@ -23,7 +23,7 @@ We explicitly test NGINX Ingress Controller on a range of Kubernetes platforms f
 {{< bootstrap-table "table table-bordered table-striped table-responsive" >}}
 | NIC Version | Supported Kubernetes Version | NIC Helm Chart Version | NIC Operator Version | NGINX / NGINX Plus version |
 | --- | --- | --- | --- | --- |
-| 3.3.2 | 1.28 - 1.22 | 1.0.2 | 2.0.2 | 1.25.3 / R30 |
+| 3.4.0 | 1.28 - 1.22 | 1.0.2 | 2.0.2 | 1.25.3 / R30 |
```

Comment:
this should be appending the upcoming release instead of replacing

Reply:
@vepatel should be updated now